### ISSUE TITLE
2019 - Regarding directory structure

### ISSUE THREAD

**Issue by yeban** [wurmlab/genomicscourse#40]
Everyone finds the directory structure hard to follow. Two ideas that have come up during discussions:

1. Provide a tree output similar to practical 1 & 4 (i.e., read cleaning and variant calling) in practicals 2 & 3 so that students have a reference.
2. Skip the first level of hierarchy. Instead, students create 01-read_cleaning, 02-assembly, and so on directly in their home directory. Of course, we will have timestamp instead of 01, 02, etc. in this case.

Option 2 is likely simpler for students, but likely a bit more work from our end because things will have to be retested. Also, it encourages more modular analysis.

Option 1 keeps the current setup and is less work (no retesting required).

**Comment by yannickwurm** [comment]
Lets go for option 2 as we're just introducing them to things

e,g. 
~/2019-09-30-mapping_to_reference/  can contain real directories;
 input (real directory containing links to elsewhere)
 results
 tmp

**Comment by yeban** [comment]
This has been addressed for practicals 1, 2, and 3.

TODO: 4 & 5

**Comment by yeban** [comment]
@MPriebe took care of this for practicals 4 & 5
